queries/ql/indents.scm

```diff
@@ -1,4 +1,5 @@
 [
+  (moduleMember)
```

Comment:
could you add a test for this?

Reply:
I did, thanks!

Reply:
Actually, thanks to the tests I realize that the indentation is not yet ok. Will fix and add more tests

Reply:
Ok, seems like its working now! BTW, I tried running the `scripts/run_tests.sh` script but it seems to be missing the `plugin/plenary.vim` file. Is that correct?